effects/populate-cache/default.nix

```diff
@@ -0,0 +1,179 @@
+{inputs, withSystem, ...}:
+let
+  attic-client = inputs.attic.packages."x86_64-linux".attic-client;
```

Comment:
Why here is hardcoded `x86_64-linux` system, shouldn't we use herculesCI.ciSystems here?

Reply:
Maybe, because I dont understand how system plays with effects.

@roberth Could you help? Am I free to use any system I find convenient for effect definition? What happens if I mention `darwin` derivation paths in my `linux` (`linux` in a sense that it uses `withSystem "linux" ({hci-effects}: ...)`) effect?
Specifically here, do I need separate effects for pushing paths for different systems to cache?

Reply:
Unresolved. Moved to: https://github.com/hercules-ci/hercules-ci-effects/pull/165/files#r1472198232